src/Advection/momentum_advection_operators.jl

```diff
@@ -77,11 +77,11 @@ end
 """
     div_Uw(i, j, k, grid, advection, U, w)
 
-Calculate the advection of momentum in the z-direction using the conservative form, ∇·(Uw)
+Calculate the advection of momentum in the ``z``-direction using the conservative form, ``𝛁⋅(𝐯 w)``,
```

Comment:
This isn't a captial w is it?  I

Reply:
nope, lowercase

Reply:
Thanks.  Clearly I need to get stronger glasses.